roles/openshift_descheduler/README.md

```diff
@@ -0,0 +1,31 @@
+Openshift descheduler
+=====================
+
+Install the descheduler
+
+Role Variables
+--------------
+Check defaults/main.yml
+
+
+Example Playbook
+----------------
+
+#!/usr/bin/ansible-playbook
+
+Notes
+-----
+
+This is currently experimental software.  This role allows users to install the descheduler and the necessary authorization pieces that allow the descheduler to function.
+
+https://github.com/kubernetes-incubator/descheduler
```

Comment:
Just asking if the `kubernetes-incubator` is what we want to present here rather than `openshift`?

Reply:
I will change to openshift/descheduler.